README.md

```diff
@@ -13,7 +13,7 @@ The Indel workflow was in the pan-cancer analysis of whole genomes (PCAWG) and c
   Nature volume 578, pages 82–93 (2020).<br>
   DOI [10.1038/s41586-020-1969-6](https://doi.org/10.1038/s41586-020-1969-6)
 
-Containers are available in [Dockstore](https://dockstore.org/containers/quay.io/pancancer/pcawg-dkfz-workflow)
+Containers are available in [Dockstore](https://dockstore.org/containers/quay.io/pancancer/pcawg-dkfz-workflow).
```

Comment:
Do you know which version of the indel workflow is present in this docker image? I can't figure it out. Users might think different versions will be available here.

Reply:
Hm, very good point. I will check.

Reply:
Here are the versions: https://github.com/ICGC-TCGA-PanCancer/dkfz_dockered_workflows/blob/00aeec8fb45ce14a3682298ade4f6a625ac5e9ad/runwrapper.sh#L112

Reply:
Thanks, @vinjana. Looks like it is a pre-Git version from the SVN era.

Reply:
Yes. It's a version in which the indel was still, together with all the other workflows, part of the COWorkflowsPlugin. This has been changed already in 2015, though.

Reply:
@NagaComBio I updated the comment with a reference to a line in the container wrapper that states the version.